src/lib/http/error.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package error
+package http
```

Comment:
Putting it under the package `lib/errors` is more reasonable as most of the logic is assembling errors?

Reply:
And please move the definition of `ErrResponder` to `src/server/v2.0/handler/base.go`, it isn't a common model